src/z3c/pt/pagetemplate.py

```diff
@@ -283,6 +278,9 @@ class BoundPageTemplate(object):
     the class instance on access, which is implemented using this
     helper class."""
 
+    im_self = None
+    im_func = None
```

Comment:
Bound page templates mimic bound Python methods, which raises questions -- shouldn't they use `__self__` and `__func__` on Python 3, or at least provide these as aliases?

( `__self__` and `__func__` have been aliases for `im_self` and `im_func` since Python 2.6)

What does zope.pagetemplate do?

Reply:
Ok, turns out the classes I was thinking about are currently in [zope.browserpage])(https://github.com/zopefoundation/zope.browserpage/blob/master/src/zope/browserpage/viewpagetemplatefile.py#L68), and they use `__self__` and `__func__`.

Reply:
I'm happy to add aliases. z3c.template would need the same treatment.

I'll open a separate PR to do that.